src/packages/@ncigdc/modern_components/BiospecimenCard/BioTreeView.js

```diff
@@ -25,8 +31,13 @@ const BioTreeView = ({
   query,
   childrenExpanded,
 }) => {
+  const entityTypes = ['portions', 'analytes', 'aliquots', 'slides'];
   const expanded =
-    ex || (query && entities.hits.edges.some(e => search(query, e).length));
+    ex ||
+    (query &&
+      (entities.hits.edges.some(e => search(query, e).length) ||
+        entityTypes.find(t => t.includes(query)) ||
```

Comment:
this is cool could you lower case query so it works for `Slides` as well as `slides`